user-guide/src/scientists/installation.md

```diff
@@ -71,8 +71,7 @@ If you see a version number, the installation was successful; but if you see an
 ## Compiling the binary
 Instead of downloading the binary and running it, you can also choose to compile the binary yourself. This is usually only necessary if you need a cutting-edge latest, unreleased version, you have an OS or processor architecture for which there is no `brane`-executable readily available or you are actively developing the framework.
 
-To compile the binary, we refer your to the [installation chapter for software engineers](../software-engineers/installation.md#compiling-the-binary), which contains information on how to do exactly this.
-
+To compile the binary, refer to the [compilation instructions](https://wiki.enablingpersonalizedinterventions.nl/specification/setup/compilation.html) over at the [Brane: A Specification](https://wiki.enablingpersonalizedinterventions.nl/specification)-book for instructions.
```

Comment:
I think it would be nicer to use relative URLs. Also the same error with setup vs development